helm/README.md

```diff
@@ -189,12 +193,33 @@ You may optionally deploy our lightweight Quorum Explorer, which is compatible f
 
 **Note:** It will be necessary to update the `quorum-explorer-config` configmap after deployment to provide the application endpoints to the nodes on the network. You may choose to either use internal k8s DNS or through ingress (your preference and needs). Please see the `values/explorer.yaml` to see some examples (you can uncomment the Besu `explorerConfig`) and how to extend the configuration yourself.
```

Comment:
Just a tiny nitpick - change `values/explorer.yaml` to `values/explorer-<client>.yaml` or something along those lines